example/_config.yml

```diff
@@ -9,6 +9,15 @@ baseurl: "/minima"
 twitter_username: jekyllrb
 github_username:  jekyll
 
+### Basic localisation (l10n)
+# If you customize the lang parameter, minima will try to use the strings in _data/locales/[lang].yml
+# Read https://github.com/jekyll/minima/README.md for more information.
+#lang: fr
```

Comment:
I prefer comments to **always** have a whitespace after the `#`. Looks better.  
(Plus, it doesnt seem like a reference-tag.. 
like how we would link to a heading in HTML e.g. `/about.html#work-experience` though I agree, its of no significance here.)